cfme/infrastructure/provider/__init__.py

```diff
@@ -165,9 +165,8 @@ def _form_mapping(self, create=None, **kwargs):
     @cached_property
     def vm_name(self):
         return version.pick({
-            version.LOWEST: "VMs",
-            '5.5': "VMs and Instances",
-            '5.7': "Virtual Machines"})
+            version.LOWEST: "VMs and Instances",
+            '5.7.1': "Virtual Machines"})
```

Comment:
As I already answered to comment from mshriver, in 5.7.0 it's actually still "VMs and Instances" and someone changed it to "Virtual Machines" for all 5.7 versions, which caused automation failed on latest 5.7.0 version. The string was changed to "Virtual Machines" only in version 5.7.1. not 5.7.0.